charts/umbrella/Chart.yaml

```diff
@@ -75,22 +70,32 @@ dependencies:
     # TX Data Consumer 1
   - name: tx-data-provider
     alias: dataconsumerOne
-    version: 0.0.5
-    repository: https://eclipse-tractusx.github.io/charts/dev
+    version: 0.1.0
+    repository: file://../tx-data-provider
     condition: dataconsumerOne.enabled
     # TX Data Providers
   - name: tx-data-provider
-    version: 0.0.5
-    repository: https://eclipse-tractusx.github.io/charts/dev
+    version: 0.1.0
+    repository: file://../tx-data-provider
     condition: tx-data-provider.enabled
     # TX Data Consumer 2
   - name: tx-data-provider
     alias: dataconsumerTwo
-    version: 0.0.5
-    repository: https://eclipse-tractusx.github.io/charts/dev
+    version: 0.1.0
+    repository: file://../tx-data-provider
     condition: dataconsumerTwo.enabled
     # pgadmin4 as helper tool for easy database access
   - condition: pgadmin4.enabled
     name: pgadmin4
     repository: https://helm.runix.net
     version: 1.25.x
+    # bdrs
+  - name: bdrs-server-memory
```

Comment:
For easier configuration, I went with the in-memory version and for now I would prefer to stick with the in-memory variant for the time being with the possibility to change to the persistent variant in the future. Especially since the BDRS data is seeded at start-up, so no initial configuration is required.